Jenkinsfile_CNP

```diff
@@ -106,9 +106,9 @@ withPipeline("nodejs", product, component) {
   before('functionalTest:preview') {
     yarnBuilder.yarn('setup')
 
-    env.COS_API_URL="http://cos-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
-    env.AAC_API_URL="http://aac-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
-    env.CCD_DATA_API_URL="http://ccd-data-store-api-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.COS_API_URL="https://cos-finrem-ccd-definitions-pr-${CHANGE_ID}.service.core-compute-preview.internal"
```

Comment:
```suggestion
    env.COS_API_URL="http://finrem-ccd-definitions-pr-830-finrem-cos"
```

Reply:
This seems to be a URL used for svc to svc communication, it has to be svc url

Reply:
applied change